includes/admin/admin-bar.php

```diff
@@ -19,48 +19,126 @@
  */
 function edd_maybe_add_store_mode_admin_bar_menu( $wp_admin_bar ) {
 
-	// Bail if no admin bar
+	// Bail if no admin bar.
 	if ( empty( $wp_admin_bar ) ) {
 		return;
 	}
 
-	// Bail if user cannot manage shop settings
+	// Bail if user cannot manage shop settings.
 	if ( ! current_user_can( 'manage_shop_settings' ) ) {
 		return;
 	}
 
-	// String
+	$indicator     = '';
+	$notifications = EDD()->notifications->countActiveNotifications();
+
+	if ( $notifications ) {
+		$count     = $notifications < 10 ? $notifications : '!';
+		$indicator = ' <div class="wp-core-ui wp-ui-notification edd-menu-notification-counter">' . $count . '</div>';
+	}
+
+	// Add the menu
+	$wp_admin_bar->add_menu(
+		array(
+			'id'    => 'edd-store-menu',
+			'title' => 'EDD' . $indicator,
+			'href'  => edd_get_admin_url(
+				array(
+					'page' => 'edd-reports',
+				)
+			),
+		)
+	);
+
+	$wp_admin_bar->add_menu( array(
```

Comment:
Since the top/parent menu item also goes to reports, is it redundant/necessary to have this as well? Wondering if the top should go somewhere else (main downloads list maybe) instead? Possibly the top one could not be a link, but that's problematic if it's showing that there are notifications.